secator/query/json.py

```
@@ -126,45 +132,36 @@ def _load_all_findings(self) -> List[Dict[str, Any]]:
 						runner_id = report_dir.name
-
-						for type_name, items in results.items():
-							if isinstance(items, list):
-								for item in items:
-									# Inject runner context from directory path if not already present
-									if f'{runner_type_singular}_id' not in item['_context']:
-										item['_context'][f'{runner_type_singular}_id'] = runner_id
-								findings.extend(items)
+						with open(report_file, 'r') as f:
+							data = json_stream.load(f)
+							try:
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)